### PR TITLE
Fixes Issue #2083 (enable vertex colors in glTF loader)

### PR DESCRIPTION
**Pull request by codex128**
Fixes enabling of vertex colors in the gltf loader as described in #2083.

### CODE REVIEW

File: jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
```
Comment (stephengold) [inline jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:1]: Please include the standard JME license header.

File: jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * Tests issue 2083.
+ * 
+ * Vertex colors on gltf models was not being enabled.
+ * 
+ * @author codex
+ */
+public class TestIssue2083 extends SimpleApplication {
+    
+    public static void main(String[] args) {
```
Comment (stephengold) [inline jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:21]: Public method should have javadoc

File: jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+import com.jme3.app.SimpleApplication;
+import com.jme3.light.DirectionalLight;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Spatial;
+
+/**
+ * Tests issue 2083.
+ * 
+ * Vertex colors on gltf models was not being enabled.
```
Comment (stephengold) [inline jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:15]: Test documentation should indicate how to determine whether the test passed or failed.

**Review by stephengold — CHANGES_REQUESTED** [review]

File: jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java (hunk 1 above)
Comment (codex128) [reply jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:1]: I copied the header from another class. What should I put for the date on the copyright?

Comment (stephengold) [reply jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:1]: Assuming this is newly-written code, "2023" will do fine for the copyright date.

File: jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+ * If the test passes, the cube should colored different hues.
+ * If the test failes, the cube will be shaded white all over.
+ * 
+ * @author codex
+ */
+public class TestIssue2083 extends SimpleApplication {
+    
+    /**
+     * Launches the application.
+     * @param args
```
Comment (stephengold) [inline jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java:51]: @param tag shouldn't appear with an empty description.

**Review by stephengold — APPROVED** [review]

**Comment by stephengold** [comment]
Thank you, @codex128 !

Unless there's additional substantive feedback, I plan to integrate this PR in about 48 hours.

**Comment by stephengold** [comment]
Ugh! Huge merge conflict with PR #2104. Could someone resolve it for me?

**Comment by riccardobl** [comment]
> Ugh! Huge merge conflict with PR #2104. Could someone resolve it for me?

Working on it

**Comment by riccardobl** [comment]
> Ugh! Huge merge conflict with PR #2104. Could someone resolve it for me?

Resolved and added a test.

**Comment by stephengold** [comment]
Thank you for your contributions, @codex128 and @riccardobl .

**Comment by stephengold** [comment]
FoxCC has reported a possible issue with e440f31:  https://hub.jmonkeyengine.org/t/jmonkeyengine-v3-7-0-beta1-release/47653/40

**Pull request merged**